.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
```

Comment:
I'm guessing the webkit stuff isn't working because it would only run if you add a `macos-latest` run to the matrix where Safari would be available on the runner

Reply:
That sounds reasonable.  I'll let you or others experiment after I land this PR.